README.rst

```diff
@@ -0,0 +1,34 @@
+What is Kolibri?
+----------------
+
+Kolibri is an adaptable measured learning environment. A user can install Kolibri and serve the app on a local network,
```

Comment:
Rather than being an LMS, isn't Kolibri primarily about distributing interactive learning resources for offline usage? LMS sounds a lot like old days and Moodle, maybe I'm too prejudist? :)